event-manager/src/main/java/evrentan/community/eventmanager/repository/EventParticipantRepository.java

```diff
@@ -0,0 +1,18 @@
+package evrentan.community.eventmanager.repository;
+
+
+import evrentan.community.eventmanager.entity.EventParticipant;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.UUID;
+
+/**
+ * Repository for EventParticipant
+ *
+ * @author <a href="https://github.com/Onuraktasj">Onur Aktas</a>
+ * @since 1.0.0
+ */
+@Repository
+public interface EventParticipantRepository extends JpaRepository<EventParticipant, UUID> {
+}
```

Comment:
please check the coding standard here. you put empty lines for some classes but do not put for others.